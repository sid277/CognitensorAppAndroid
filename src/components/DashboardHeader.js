import React from 'react';
import { Text, StyleSheet, View } from 'react-native';
import { theme } from '../theme';

const DashboardHeader = ({ title }) => {
  return (
    <View style={styles.header}>
      <Text style={styles.headerFont}>{title}</Text>
    </View>
  );
};

const styles = StyleSheet.create({
  header: {
    padding: theme.spacing.small,
  },
  headerFont: {
    ...theme.typography.header,
  },
});

export default DashboardHeader;
