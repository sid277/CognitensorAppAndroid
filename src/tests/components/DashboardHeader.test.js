import 'react-native';
import React from 'react';
import DashboardHeader from '../../components/DashboardHeader';
import renderer from 'react-test-renderer';

it('renders DashboardHeader component', () => {
  const tree = renderer.create(<DashboardHeader />).toJSON();
  expect(tree).toMatchSnapshot();
});
