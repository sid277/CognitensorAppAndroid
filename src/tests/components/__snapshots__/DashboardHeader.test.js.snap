// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders DashboardHeader component 1`] = `
<View
  style={
    Object {
      "padding": 16,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 18,
        "letterSpacing": 0.5,
      }
    }
  />
</View>
`;
